data/ch/about.yml

```diff
@@ -0,0 +1,16 @@
+############################# About ###################################
+about:
+  enable : true
+  title : About Us
+  about_item :
+  - icon : ti-world # themify icon pack : https://themify.me/themify-icons
+    title : Fournir des données distribuées
+    content : Notre spécialité est la construction et la maintenance de services de données distribuées à l'échelle planétaire.
+
+  - icon : ti-link # themify icon pack : https://themify.me/themify-icons
+    title : Connecter les communautés mondiales
+    content : Nous respectons les droits légaux, la vie privée et les normes culturelles de nos utilisateurs dans le monde entier.
+
+  - icon : ti-github # themify icon pack : https://themify.me/themify-icons
+    title : S'engager avec l'Open Source
+    content : Nous croyons au pouvoir de l'open source pour améliorer la compréhension, démocratiser l'accès et renforcer la fiabilité des logiciels.
```

Comment:
We can remove this file (we'll add Chinese-language in a future PR using the go-18n for Hugo approach)